docs/deploy_logging.md

```diff
@@ -0,0 +1,53 @@
+# Deploy openshift-logging on cluster
```

Comment:
I am missing the link to this documentation from some other place? I think there should b e link in getting started section to this and also from this file: https://github.com/red-hat-storage/ocs-ci/blob/master/docs/README.md

Reply:
done